src/style.css

```diff
@@ -1 +1,67 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap");
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
I prefer having this rule where it is required but not for all the tags

Reply:
yes, it is a habbit I developed.. will change it